app-starters-common-analytics/src/main/java/org/springframework/cloud/stream/app/metrics/memory/InMemoryAggregateCounterRepository.java

```diff
@@ -70,6 +74,16 @@ public AggregateCounter getCounts(String name, Interval interval, AggregateCount
 	}
 
 	@Override
+	public AggregateCounter findOne(String name) {
+		return counters.get(name);
+	}
+
+	@Override
+	public Collection<String> list() {
+		return aggregates.keySet();
```

Comment:
Should we ensure for better usability that the returned collection is sorted `ascending` by name?